ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -301,24 +305,50 @@ const PANELS = PANEL_DETAILS.map(panelDetails)
 const STAFF_PANELS = STAFF_PANEL_DETAILS.map(panelDetails)
 
 class VariantSearchFormContent extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { activeIndex: [] }
+  }
+
+  expandAll = () => {
+    this.setState({ activeIndex: [0, 1, 2, 3, 4, 5] })
+  }
+
+  collapseAll = () => {
+    this.setState({ activeIndex: [] })
+  }
+
+  handleTitleClick = (e, itemProps) => {
+    const { index } = itemProps
```

Comment:
A neat syntax you can use in es6 combines the above 2 lines as follows:
`handleTitleClick = (e, { index }) => {`